pkg/fake/cloudprovider.go

```diff
@@ -72,6 +71,10 @@ func (c *CloudProvider) Get(context.Context, string, string) (*v1alpha5.Machine,
 	return nil, nil
 }
 
+func (c *CloudProvider) HydrateMachine(_ context.Context, _ *v1alpha5.Machine) error {
+	return nil
+}
```

Comment:
Can't we make this part of the AWS API, but exclude it from other providers?

Reply:
That's what the current state is. We have this only in the AWS API but not a part of the generic interface.

Reply:
Why include it in the fake, then?

Reply:
Good point, that's an artifact of an older commit. I'll remove.